civihr_employee_portal/civihr_employee_portal.install

```diff
@@ -276,6 +276,22 @@ function civihr_employee_portal_update_7016() {
   drupal_write_record('webform_civicrm_forms', $civicrmWebform, 'nid');
 }
 
+/**
+ * Remove old and unused links from the Reports menu
+ */
+function civihr_employee_portal_update_7017() {
```

Comment:
In the PR description it says

> This was making the features module to consider them as two different menu links. To fix this issue, on a1142fe the old reference to the main-menu_civihr-admin:civicrm was removed and a new one was added to main-menu_hr-admin:civicrm. 

but I don't see it being removed

Reply:
You can see that here: https://github.com/compucorp/civihr-employee-portal/commit/a1142fe022674dd3ef5f34648febd30c80fae6fe#diff-8b8c940cf502b4f81339dc0f35970dcaL71

And here is where it gets added with the new identifier: https://github.com/compucorp/civihr-employee-portal/commit/a1142fe022674dd3ef5f34648febd30c80fae6fe#diff-8b8c940cf502b4f81339dc0f35970dcaR135

Here it's removed from the .info file: https://github.com/compucorp/civihr-employee-portal/commit/a1142fe022674dd3ef5f34648febd30c80fae6fe#diff-b7976a3bd66a605a801c4cb57103e1ebL53

And here the new one is being added: https://github.com/compucorp/civihr-employee-portal/commit/a1142fe022674dd3ef5f34648febd30c80fae6fe#diff-b7976a3bd66a605a801c4cb57103e1ebR53

Reply:
Ah ok, I don't really know how the menu links are all working so I thought maybe it needed a `db_delete` like the ones below too, but if not then that's fine.